.github/workflows/pull-request.yml

```
@@ -419,22 +417,119 @@ jobs:
+          if [ -n "$(git status --porcelain)" ]; then
+            echo "::error::Schema files are outdated or missing."
+            echo "Please run 'pnpm docusaurus graphql-to-doc' inside '/docs' locally and commit the updated files."
+            echo ""
+            echo "Changed files:"
+            git status --porcelain
+            exit 1
+          else
+            echo "Schema is up to date."
+          fi
+
+  Pre-Test-Checks-Pass:
```

actionlint (1.7.9)

job "pre-test-checks-pass" needs job "check-graphql-tada" which does not exist in this workflow

(job-needs)